docs/deployment/shared_data/minio.md

```diff
@@ -0,0 +1,474 @@
+
+The S3 path used to store data. It consists of the name of your S3 bucket and the sub-path (if any) under it, for example, `testbucket/subpath`.
+
+#### aws_s3_endpoint
+
+The endpoint used to access your S3 bucket, for example, `https://s3.us-west-2.amazonaws.com`.
+
+#### aws_s3_region
+
+The region in which your S3 bucket resides, for example, `us-west-2`.
+
+#### aws_s3_use_aws_sdk_default_behavior
+
+Whether to use the [AWS SDK default credentials provider chain](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html). Valid values:
+
+- `true`
+- `false` (Default).
+
+#### aws_s3_use_instance_profile
+
+Whether to use Instance Profile and Assumed Role as credential methods for accessing S3. Valid values:
+
+- `true`
+- `false` (Default).
+
+If you use IAM user-based credential (Access Key and Secret Key) to access S3, you must specify this item as `false`, and specify `aws_s3_access_key` and `aws_s3_secret_key`.
+
+If you use Instance Profile to access S3, you must specify this item as `true`.
+
+If you use Assumed Role to access S3, you must specify this item as `true`, and specify `aws_s3_iam_role_arn`.
+
+And if you use an external AWS account,  you must also specify `aws_s3_external_id`.
+
+#### aws_s3_access_key
+
+The Access Key ID used to access your S3 bucket.
+
+#### aws_s3_secret_key
+
+The Secret Access Key used to access your S3 bucket.
+
+#### aws_s3_iam_role_arn
+
+The ARN of the IAM role that has privileges on your S3 bucket in which your data files are stored.
+
+#### aws_s3_external_id
+
+The external ID of the AWS account that is used for cross-account access to your S3 bucket.
+
+#### azure_blob_path
+
+The Azure Blob Storage path used to store data. It consists of the name of the container within your storage account and the sub-path (if any) under the container, for example, `testcontainer/subpath`.
+
+#### azure_blob_endpoint
+
+The endpoint of your Azure Blob Storage Account, for example, `https://test.blob.core.windows.net`.
+
+#### azure_blob_shared_key
+
+The Shared Key used to authorize requests for your Azure Blob Storage.
+
+#### azure_blob_sas_token
+
+The shared access signatures (SAS) used to authorize requests for your Azure Blob Storage.
+
+> **Note**
+>
+> Only credential-related configuration items can be modified after your shared-data StarRocks cluster is created. If you changed the original storage path-related configuration items, the databases and tables you created before the change become read-only, and you cannot load data into them.
+
+If you want to create the default storage volume manually after the cluster is created, you only need to add the following configuration items:
+
+```Properties
+run_mode = shared_data
+cloud_native_meta_port = <meta_port>
+enable_load_volume_from_conf = false
+```
+
+If you want to specify the properties of your object storage in the FE configuration file, examples are as follows:
+
+- If you use AWS S3
```

Comment:
do we need to remove all the rest configurations but just keep the minIO related content, because this doc is specific for minIO case.

Reply:
yes, sorry: I have not finished editing the MinIO or Azure docs yet.